cluster/examples/kubernetes/ceph/kube-registry.yaml

```diff
@@ -1,3 +1,13 @@
+---
+# Create a service account for the kube registry limited to the kube-system namespace
+# pod security policies can be bound to this service account if necessary
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-registry
+  namespace: kube-system
+---
+# Create the HA registry
 apiVersion: v1
 kind: ReplicationController
```

Comment:
Could you update the example from `ReplicationController` to `Deployment` while you are at it?